scripts/clean_up.py

```diff
@@ -0,0 +1,121 @@
+""" This script cleans up the sequence data and metadata within GCS and Metamist """
+
+import logging
+import coloredlogs
+import click
+
+from cloudpathlib import CloudPath
+from google.cloud import storage
+
+from sample_metadata.apis import SampleApi, SequenceApi
+from sample_metadata.models import (
+    SequenceUpdateModel,
+)
+
+
+client = storage.Client()
+sapi = SampleApi()
+seqapi = SequenceApi()
+coloredlogs.install()
+
+
+def clean_up_metamist(
+    sequence_ids_internal: list[str],
+    clean_up_location: str,
+):
+    """Deleting sequencing entires for specified sequences in metamist
+    This will delete all of the reads for the latest sequence."""
+    all_locations: list[CloudPath] = []
+    sequence_ids: list[int] = []
+    for sequence_id in sequence_ids_internal:
+        current_sequence = seqapi.get_sequence_by_id(sequence_id)
+        reads = current_sequence['meta']['reads']
+        current_read_locations: list[CloudPath] = []
+        # Skip delete operations if there are no reads for the given sequence
+        delete_reads = bool(reads)
+        for read in reads:
```

Comment:
Reads can be both crams/bams and fastqs and they have a slightly different structure.

Reply:
Added a check for now so we only support fastqs. Can expand this in future if there is a use case.